decidim-demographics/app/packs/stylesheets/decidim/demographics/demographics.scss

```
@@ -0,0 +1,39 @@
+.sociodemographic.response-questionnaire {
+  @apply border-t-0 border-background pt-8;
+}
+
+.sociodemographic {
+  .response-questionnaire {
+    &__question-label {
```

Can we consolidate these rules and also change the main class to `demographic`? I didn't realize in my first review the usage of both terms, sorry!! 

```suggestion
.demographic {
  .response-questionnaire {
    @apply border-t-0 border-background pt-8;

    &__question-label {
```

---

This is not working as you put it. I have added the demographic css class on the same level as `response-questionnaire`. If we apply the suggestion, it will assume that `response-questionnaire__question-label` will also need to have the `demographic` class which is not the case.

---

fixed in 3aa484f64a.
As said we cannot consolidate it as you suggested.